[2025-06-04 15:32:36,777 : INFO : main : Welcome to our Project]
[2025-06-05 12:01:19,405 : INFO : main : >>>>> stage DATA INGESTION STAGE started <<<<<]
[2025-06-05 12:01:19,419 : INFO : shared : Yaml File: config\config.yaml loaded successfully!]
[2025-06-05 12:01:19,435 : INFO : shared : Yaml File: params.yaml loaded successfully!]
[2025-06-05 12:01:19,435 : INFO : shared : Yaml File: schema.yaml loaded successfully!]
[2025-06-05 12:01:19,436 : ERROR : main : Yaml file schema.yaml is empty!]
Traceback (most recent call last):
  File "C:\ML_OPS\Projects\project1\src\DataScienceProject\utils\shared.py", line 28, in read_yaml
    return ConfigBox(content)
  File "box\\box.py", line 295, in box.box.Box.__init__
box.exceptions.BoxValueError: First argument must be mapping or iterable

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\ML_OPS\Projects\project1\main.py", line 7, in <module>
    data_ingestion.initiate_data_ingestion()
  File "C:\ML_OPS\Projects\project1\src\DataScienceProject\pipeline\data_ingestion_pipeline.py", line 11, in initiate_data_ingestion
    config = ConfigurationManager()
  File "C:\ML_OPS\Projects\project1\src\DataScienceProject\config\configuration.py", line 13, in __init__
    self.schema=read_yaml(schema_filepath)
  File "C:\ML_OPS\Projects\project1\myvenv\lib\site-packages\ensure\main.py", line 872, in __call__
    return_val = self.f(*args, **kwargs)
  File "C:\ML_OPS\Projects\project1\src\DataScienceProject\utils\shared.py", line 30, in read_yaml
    raise ValueError(f'Yaml file {path_to_yaml} is empty!')
ValueError: Yaml file schema.yaml is empty!
[2025-06-05 12:02:41,389 : INFO : main : >>>>> stage DATA INGESTION STAGE started <<<<<]
[2025-06-05 12:02:41,389 : INFO : shared : Yaml File: config\config.yaml loaded successfully!]
[2025-06-05 12:02:41,390 : INFO : shared : Yaml File: params.yaml loaded successfully!]
[2025-06-05 12:02:41,392 : INFO : shared : Yaml File: schema.yaml loaded successfully!]
[2025-06-05 12:02:41,392 : ERROR : main : Argument path_to_directories of type <class 'str'> to <function create_directories at 0x0000020C3323CA60> does not match annotation type <class 'list'>]
Traceback (most recent call last):
  File "C:\ML_OPS\Projects\project1\main.py", line 7, in <module>
    data_ingestion.initiate_data_ingestion()
  File "C:\ML_OPS\Projects\project1\src\DataScienceProject\pipeline\data_ingestion_pipeline.py", line 11, in initiate_data_ingestion
    config = ConfigurationManager()
  File "C:\ML_OPS\Projects\project1\src\DataScienceProject\config\configuration.py", line 15, in __init__
    create_directories(self.config.artifacts_root)
  File "C:\ML_OPS\Projects\project1\myvenv\lib\site-packages\ensure\main.py", line 833, in __call__
    raise EnsureError(msg.format(arg=arg, f=self.f, t=templ, valt=type(value)))
ensure.main.EnsureError: Argument path_to_directories of type <class 'str'> to <function create_directories at 0x0000020C3323CA60> does not match annotation type <class 'list'>
[2025-06-05 12:04:48,927 : INFO : main : >>>>> stage DATA INGESTION STAGE started <<<<<]
[2025-06-05 12:04:48,928 : INFO : shared : Yaml File: config\config.yaml loaded successfully!]
[2025-06-05 12:04:48,929 : INFO : shared : Yaml File: params.yaml loaded successfully!]
[2025-06-05 12:04:48,929 : INFO : shared : Yaml File: schema.yaml loaded successfully!]
[2025-06-05 12:04:48,930 : INFO : shared : created directory at : artifacts]
[2025-06-05 12:04:48,930 : INFO : shared : created directory at : articacts/data_ingestion]
[2025-06-05 12:04:48,939 : ERROR : main : "'ConfigBox' object has no attribute 'source'"]
Traceback (most recent call last):
  File "box\\box.py", line 594, in box.box.Box.__getitem__
KeyError: 'source'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "box\\box.py", line 633, in box.box.Box.__getattr__
  File "box\\box.py", line 621, in box.box.Box.__getitem__
box.exceptions.BoxKeyError: "'source'"

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "box\\box.py", line 635, in box.box.Box.__getattr__
AttributeError: 'ConfigBox' object has no attribute 'source'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "box\\config_box.py", line 29, in box.config_box.ConfigBox.__getattr__
  File "box\\box.py", line 649, in box.box.Box.__getattr__
box.exceptions.BoxKeyError: "'ConfigBox' object has no attribute 'source'"

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "box\\box.py", line 594, in box.box.Box.__getitem__
KeyError: 'source'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "box\\box.py", line 633, in box.box.Box.__getattr__
  File "box\\box.py", line 621, in box.box.Box.__getitem__
box.exceptions.BoxKeyError: "'source'"

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "box\\box.py", line 635, in box.box.Box.__getattr__
AttributeError: 'ConfigBox' object has no attribute 'source'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\ML_OPS\Projects\project1\main.py", line 7, in <module>
    data_ingestion.initiate_data_ingestion()
  File "C:\ML_OPS\Projects\project1\src\DataScienceProject\pipeline\data_ingestion_pipeline.py", line 12, in initiate_data_ingestion
    data_ingestion_config = config.get_data_ingestion_config()
  File "C:\ML_OPS\Projects\project1\src\DataScienceProject\config\configuration.py", line 22, in get_data_ingestion_config
    source_url = config.source.url,
  File "box\\config_box.py", line 31, in box.config_box.ConfigBox.__getattr__
  File "box\\box.py", line 649, in box.box.Box.__getattr__
box.exceptions.BoxKeyError: "'ConfigBox' object has no attribute 'source'"
[2025-06-05 12:06:13,943 : INFO : main : >>>>> stage DATA INGESTION STAGE started <<<<<]
[2025-06-05 12:06:13,944 : INFO : shared : Yaml File: config\config.yaml loaded successfully!]
[2025-06-05 12:06:13,945 : INFO : shared : Yaml File: params.yaml loaded successfully!]
[2025-06-05 12:06:13,945 : INFO : shared : Yaml File: schema.yaml loaded successfully!]
[2025-06-05 12:06:13,946 : INFO : shared : created directory at : artifacts]
[2025-06-05 12:06:13,946 : INFO : shared : created directory at : articacts/data_ingestion]
[2025-06-05 12:06:14,846 : ERROR : main : [Errno 2] No such file or directory: 'artifacts/data_ingestion/data.gzip']
Traceback (most recent call last):
  File "C:\ML_OPS\Projects\project1\main.py", line 7, in <module>
    data_ingestion.initiate_data_ingestion()
  File "C:\ML_OPS\Projects\project1\src\DataScienceProject\pipeline\data_ingestion_pipeline.py", line 14, in initiate_data_ingestion
    data_ingestion.download_file()
  File "C:\ML_OPS\Projects\project1\src\DataScienceProject\components\data_ingestion.py", line 16, in download_file
    filename,headers = request.urlretrieve(
  File "C:\Users\arunshukla\AppData\Local\Programs\Python\Python310\lib\urllib\request.py", line 251, in urlretrieve
    tfp = open(filename, 'wb')
FileNotFoundError: [Errno 2] No such file or directory: 'artifacts/data_ingestion/data.gzip'
[2025-06-05 12:07:59,419 : INFO : main : >>>>> stage DATA INGESTION STAGE started <<<<<]
[2025-06-05 12:07:59,452 : INFO : shared : Yaml File: config\config.yaml loaded successfully!]
[2025-06-05 12:07:59,453 : INFO : shared : Yaml File: params.yaml loaded successfully!]
[2025-06-05 12:07:59,454 : INFO : shared : Yaml File: schema.yaml loaded successfully!]
[2025-06-05 12:07:59,456 : INFO : shared : created directory at : artifacts]
[2025-06-05 12:07:59,457 : INFO : shared : created directory at : artifacts/data_ingestion]
[2025-06-05 12:08:00,761 : INFO : data_ingestion : artifacts/data_ingestion/data.gzip download! with following info: 
Server: nginx/1.27.5
Date: Thu, 05 Jun 2025 06:37:59 GMT
Transfer-Encoding: chunked
Connection: close

]
[2025-06-05 12:08:00,776 : ERROR : main : File is not a zip file]
Traceback (most recent call last):
  File "C:\ML_OPS\Projects\project1\main.py", line 7, in <module>
    data_ingestion.initiate_data_ingestion()
  File "C:\ML_OPS\Projects\project1\src\DataScienceProject\pipeline\data_ingestion_pipeline.py", line 15, in initiate_data_ingestion
    data_ingestion.extract_zip_file()
  File "C:\ML_OPS\Projects\project1\src\DataScienceProject\components\data_ingestion.py", line 31, in extract_zip_file
    with zipfile.ZipFile(self.config.local_data_file, 'r') as zip_ref:
  File "C:\Users\arunshukla\AppData\Local\Programs\Python\Python310\lib\zipfile.py", line 1267, in __init__
    self._RealGetContents()
  File "C:\Users\arunshukla\AppData\Local\Programs\Python\Python310\lib\zipfile.py", line 1334, in _RealGetContents
    raise BadZipFile("File is not a zip file")
zipfile.BadZipFile: File is not a zip file
[2025-06-05 12:09:15,491 : INFO : main : >>>>> stage DATA INGESTION STAGE started <<<<<]
[2025-06-05 12:09:15,506 : INFO : shared : Yaml File: config\config.yaml loaded successfully!]
[2025-06-05 12:09:15,508 : INFO : shared : Yaml File: params.yaml loaded successfully!]
[2025-06-05 12:09:15,509 : INFO : shared : Yaml File: schema.yaml loaded successfully!]
[2025-06-05 12:09:15,509 : INFO : shared : created directory at : artifacts]
[2025-06-05 12:09:15,510 : INFO : shared : created directory at : artifacts/data_ingestion]
[2025-06-05 12:09:16,840 : INFO : data_ingestion : artifacts/data_ingestion/data.zip download! with following info: 
Server: nginx/1.27.5
Date: Thu, 05 Jun 2025 06:39:15 GMT
Transfer-Encoding: chunked
Connection: close

]
[2025-06-05 12:09:16,856 : ERROR : main : File is not a zip file]
Traceback (most recent call last):
  File "C:\ML_OPS\Projects\project1\main.py", line 7, in <module>
    data_ingestion.initiate_data_ingestion()
  File "C:\ML_OPS\Projects\project1\src\DataScienceProject\pipeline\data_ingestion_pipeline.py", line 15, in initiate_data_ingestion
    data_ingestion.extract_zip_file()
  File "C:\ML_OPS\Projects\project1\src\DataScienceProject\components\data_ingestion.py", line 31, in extract_zip_file
    with zipfile.ZipFile(self.config.local_data_file, 'r') as zip_ref:
  File "C:\Users\arunshukla\AppData\Local\Programs\Python\Python310\lib\zipfile.py", line 1267, in __init__
    self._RealGetContents()
  File "C:\Users\arunshukla\AppData\Local\Programs\Python\Python310\lib\zipfile.py", line 1334, in _RealGetContents
    raise BadZipFile("File is not a zip file")
zipfile.BadZipFile: File is not a zip file
[2025-06-05 12:14:58,732 : INFO : main : >>>>> stage DATA INGESTION STAGE started <<<<<]
[2025-06-05 12:14:58,763 : INFO : shared : Yaml File: config\config.yaml loaded successfully!]
[2025-06-05 12:14:58,764 : INFO : shared : Yaml File: params.yaml loaded successfully!]
[2025-06-05 12:14:58,765 : INFO : shared : Yaml File: schema.yaml loaded successfully!]
[2025-06-05 12:14:58,766 : INFO : shared : created directory at : artifacts]
[2025-06-05 12:14:58,767 : INFO : shared : created directory at : artifacts/data_ingestion]
[2025-06-05 12:14:59,602 : INFO : data_ingestion : artifacts/data_ingestion/data.zip download! with following info: 
Connection: close
Content-Length: 23329
Cache-Control: max-age=300
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
Content-Type: application/zip
ETag: "c69888a4ae59bc5a893392785a938ccd4937981c06ba8a9d6a21aa52b4ab5b6e"
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
X-GitHub-Request-Id: 4613:204B32:1626C1:30EFE1:68413CEA
Accept-Ranges: bytes
Date: Thu, 05 Jun 2025 06:44:59 GMT
Via: 1.1 varnish
X-Served-By: cache-bom4744-BOM
X-Cache: MISS
X-Cache-Hits: 0
X-Timer: S1749105899.793317,VS0,VE350
Vary: Authorization,Accept-Encoding
Access-Control-Allow-Origin: *
Cross-Origin-Resource-Policy: cross-origin
X-Fastly-Request-ID: 21aa43e5724d4afceb66e9a6e9092c7b052e1757
Expires: Thu, 05 Jun 2025 06:49:59 GMT
Source-Age: 0

]
[2025-06-05 12:14:59,620 : INFO : data_ingestion : Zip file extracted to artifacts/data_ingestion]
[2025-06-05 12:14:59,621 : INFO : main : >>>>> stage DATA INGESTION STAGE completed! <<<<<]
[2025-06-05 17:02:55,443 : INFO : shared : Yaml File: config\config.yaml loaded successfully!]
[2025-06-05 17:02:55,460 : INFO : shared : Yaml File: params.yaml loaded successfully!]
[2025-06-05 17:02:55,476 : INFO : shared : Yaml File: schema.yaml loaded successfully!]
[2025-06-05 17:03:50,921 : INFO : shared : Yaml File: config\config.yaml loaded successfully!]
[2025-06-05 17:03:50,926 : INFO : shared : Yaml File: params.yaml loaded successfully!]
[2025-06-05 17:03:50,932 : INFO : shared : Yaml File: schema.yaml loaded successfully!]
[2025-06-05 17:04:51,222 : INFO : shared : Yaml File: config\config.yaml loaded successfully!]
[2025-06-05 17:04:51,230 : INFO : shared : Yaml File: params.yaml loaded successfully!]
[2025-06-05 17:04:51,235 : INFO : shared : Yaml File: schema.yaml loaded successfully!]
[2025-06-05 17:04:51,238 : INFO : shared : created directory at : artifacts]
[2025-06-05 17:04:51,241 : INFO : shared : created directory at : artifacts/data_validation]
[2025-06-05 17:04:51,287 : INFO : 4252915885 : Column chlorides is not in the schema.]
[2025-06-05 17:04:51,290 : INFO : 4252915885 : Column pH is not in the schema.]
[2025-06-05 17:04:51,303 : WARNING : 4252915885 : Column 'chlordes' not found in dataset]
[2025-06-05 17:08:27,053 : INFO : shared : Yaml File: config\config.yaml loaded successfully!]
[2025-06-05 17:08:27,056 : INFO : shared : Yaml File: params.yaml loaded successfully!]
[2025-06-05 17:08:27,061 : INFO : shared : Yaml File: schema.yaml loaded successfully!]
[2025-06-05 17:08:27,063 : INFO : shared : created directory at : artifacts]
[2025-06-05 17:08:27,063 : INFO : shared : created directory at : artifacts/data_validation]
[2025-06-05 17:08:27,071 : INFO : 4252915885 : Column chlorides is not in the schema.]
[2025-06-05 17:08:27,077 : INFO : 4252915885 : Column pH is not in the schema.]
[2025-06-05 17:08:27,086 : WARNING : 4252915885 : Column 'chlordes' not found in dataset]
[2025-06-05 17:09:49,979 : INFO : shared : Yaml File: config\config.yaml loaded successfully!]
[2025-06-05 17:09:49,982 : INFO : shared : Yaml File: params.yaml loaded successfully!]
[2025-06-05 17:09:49,996 : INFO : shared : Yaml File: schema.yaml loaded successfully!]
[2025-06-05 17:09:49,998 : INFO : shared : created directory at : artifacts]
[2025-06-05 17:09:49,999 : INFO : shared : created directory at : artifacts/data_validation]
[2025-06-05 17:09:58,393 : INFO : shared : Yaml File: config\config.yaml loaded successfully!]
[2025-06-05 17:09:58,395 : INFO : shared : Yaml File: params.yaml loaded successfully!]
[2025-06-05 17:09:58,400 : INFO : shared : Yaml File: schema.yaml loaded successfully!]
[2025-06-05 17:09:58,402 : INFO : shared : created directory at : artifacts]
[2025-06-05 17:09:58,405 : INFO : shared : created directory at : artifacts/data_validation]
[2025-06-05 17:38:36,367 : INFO : main : >>>>> stage DATA INGESTION STAGE started <<<<<]
[2025-06-05 17:38:36,377 : INFO : shared : Yaml File: config\config.yaml loaded successfully!]
[2025-06-05 17:38:36,387 : INFO : shared : Yaml File: params.yaml loaded successfully!]
[2025-06-05 17:38:36,407 : INFO : shared : Yaml File: schema.yaml loaded successfully!]
[2025-06-05 17:38:36,415 : INFO : shared : created directory at : artifacts]
[2025-06-05 17:38:36,415 : INFO : shared : created directory at : artifacts/data_ingestion]
[2025-06-05 17:38:38,660 : INFO : data_ingestion : artifacts/data_ingestion/data.zip download! with following info: 
Connection: close
Content-Length: 23329
Cache-Control: max-age=300
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
Content-Type: application/zip
ETag: "c69888a4ae59bc5a893392785a938ccd4937981c06ba8a9d6a21aa52b4ab5b6e"
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
X-GitHub-Request-Id: A97C:1964FB:16BA23:2FFEA0:684188C5
Accept-Ranges: bytes
Date: Thu, 05 Jun 2025 12:08:38 GMT
Via: 1.1 varnish
X-Served-By: cache-bom4722-BOM
X-Cache: MISS
X-Cache-Hits: 0
X-Timer: S1749125318.940178,VS0,VE373
Vary: Authorization,Accept-Encoding
Access-Control-Allow-Origin: *
Cross-Origin-Resource-Policy: cross-origin
X-Fastly-Request-ID: 557b1183bcdfbbac7462aa8a94d95ed2e9cf6baf
Expires: Thu, 05 Jun 2025 12:13:38 GMT
Source-Age: 0

]
[2025-06-05 17:38:38,667 : INFO : data_ingestion : Zip file extracted to artifacts/data_ingestion]
[2025-06-05 17:38:38,667 : INFO : main : >>>>> stage DATA INGESTION STAGE completed! <<<<<]
[2025-06-05 17:38:38,667 : INFO : main : >>>>> stage DATA VALIDATION STAGE started <<<<<]
[2025-06-05 17:38:38,678 : INFO : shared : Yaml File: config\config.yaml loaded successfully!]
[2025-06-05 17:38:38,678 : INFO : shared : Yaml File: params.yaml loaded successfully!]
[2025-06-05 17:38:38,678 : INFO : shared : Yaml File: schema.yaml loaded successfully!]
[2025-06-05 17:38:38,682 : INFO : shared : created directory at : artifacts]
[2025-06-05 17:38:38,682 : INFO : shared : created directory at : artifacts/data_validation]
[2025-06-05 17:38:38,729 : INFO : main : >>>>> stage DATA VALIDATION STAGE completed! <<<<<]
[2025-06-06 10:58:57,846 : INFO : shared : Yaml File: config\config.yaml loaded successfully!]
[2025-06-06 10:58:57,865 : INFO : shared : Yaml File: params.yaml loaded successfully!]
[2025-06-06 10:58:57,880 : INFO : shared : Yaml File: schema.yaml loaded successfully!]
[2025-06-06 10:58:57,883 : INFO : shared : created directory at : artifacts]
[2025-06-06 10:58:57,885 : INFO : shared : created directory at : artifacts/data_transformation]
[2025-06-06 10:58:57,940 : INFO : 107865200 : Splitted data into training and test sets]
[2025-06-06 10:58:57,941 : INFO : 107865200 : The Train Data SHAPE is: (1279, 12)]
[2025-06-06 10:58:57,942 : INFO : 107865200 : The Test Data SHAPE is: (320, 12)]
[2025-06-06 11:14:17,325 : INFO : main : >>>>> stage DATA INGESTION STAGE started <<<<<]
[2025-06-06 11:14:17,352 : INFO : shared : Yaml File: config\config.yaml loaded successfully!]
[2025-06-06 11:14:17,370 : INFO : shared : Yaml File: params.yaml loaded successfully!]
[2025-06-06 11:14:17,391 : INFO : shared : Yaml File: schema.yaml loaded successfully!]
[2025-06-06 11:14:17,392 : INFO : shared : created directory at : artifacts]
[2025-06-06 11:14:17,393 : INFO : shared : created directory at : artifacts/data_ingestion]
[2025-06-06 11:14:17,393 : INFO : data_ingestion : File Already exists]
[2025-06-06 11:14:17,431 : INFO : data_ingestion : Zip file extracted to artifacts/data_ingestion]
[2025-06-06 11:14:17,432 : INFO : main : >>>>> stage DATA INGESTION STAGE completed! <<<<<]
[2025-06-06 11:14:17,432 : INFO : main : >>>>> stage DATA VALIDATION STAGE started <<<<<]
[2025-06-06 11:14:17,434 : INFO : shared : Yaml File: config\config.yaml loaded successfully!]
[2025-06-06 11:14:17,437 : INFO : shared : Yaml File: params.yaml loaded successfully!]
[2025-06-06 11:14:17,441 : INFO : shared : Yaml File: schema.yaml loaded successfully!]
[2025-06-06 11:14:17,442 : INFO : shared : created directory at : artifacts]
[2025-06-06 11:14:17,443 : INFO : shared : created directory at : artifacts/data_validation]
[2025-06-06 11:14:17,480 : INFO : main : >>>>> stage DATA VALIDATION STAGE completed! <<<<<]
[2025-06-06 11:14:17,481 : INFO : main : >>>>> stage DATA VALIDATION STAGE started <<<<<]
[2025-06-06 11:14:17,482 : INFO : shared : Yaml File: config\config.yaml loaded successfully!]
[2025-06-06 11:14:17,482 : INFO : shared : Yaml File: params.yaml loaded successfully!]
[2025-06-06 11:14:17,484 : INFO : shared : Yaml File: schema.yaml loaded successfully!]
[2025-06-06 11:14:17,484 : INFO : shared : created directory at : artifacts]
[2025-06-06 11:14:17,485 : INFO : shared : created directory at : artifacts/data_transformation]
[2025-06-06 11:14:17,499 : INFO : data_transformation : Splitted data into training and test sets]
[2025-06-06 11:14:17,499 : INFO : data_transformation : The Train Data SHAPE is: (1279, 12)]
[2025-06-06 11:14:17,499 : INFO : data_transformation : The Test Data SHAPE is: (320, 12)]
[2025-06-06 11:14:17,500 : INFO : main : >>>>> stage DATA TRANSFORMATION STAGE completed! <<<<<]
[2025-06-06 12:06:18,486 : INFO : shared : created directory at : artifacts\test_folder]
[2025-06-06 12:10:02,996 : INFO : shared : created directory at : artifacts\test_folder]
[2025-06-06 12:10:29,482 : INFO : shared : created directory at : artifacts\test_folder]
[2025-06-06 12:25:35,162 : INFO : shared : created directory at : artifacts\test_folder]
[2025-06-06 12:32:48,229 : INFO : shared : Yaml File: config\config.yaml loaded successfully!]
[2025-06-06 12:32:48,271 : INFO : shared : Yaml File: params.yaml loaded successfully!]
[2025-06-06 12:32:48,307 : INFO : shared : Yaml File: schema.yaml loaded successfully!]
[2025-06-06 12:32:48,315 : INFO : shared : created directory at : artifacts]
[2025-06-06 12:32:48,317 : INFO : shared : created directory at : artifacts/model_trainer]
[2025-06-06 12:33:23,173 : INFO : shared : created directory at : artifacts\test_folder]
[2025-06-06 12:33:23,219 : INFO : shared : Yaml File: config\config.yaml loaded successfully!]
[2025-06-06 12:33:23,250 : INFO : shared : Yaml File: params.yaml loaded successfully!]
[2025-06-06 12:33:23,258 : INFO : shared : Yaml File: schema.yaml loaded successfully!]
[2025-06-06 12:33:23,263 : INFO : shared : created directory at : artifacts]
[2025-06-06 12:33:23,265 : INFO : shared : created directory at : artifacts/model_trainer]
[2025-06-06 12:33:23,440 : INFO : 3023829112 : Model fit training done.]
[2025-06-06 12:50:03,736 : INFO : main : >>>>> stage DATA INGESTION STAGE started <<<<<]
[2025-06-06 12:50:03,770 : INFO : shared : Yaml File: config\config.yaml loaded successfully!]
[2025-06-06 12:50:03,798 : INFO : shared : Yaml File: params.yaml loaded successfully!]
[2025-06-06 12:50:03,824 : INFO : shared : Yaml File: schema.yaml loaded successfully!]
[2025-06-06 12:50:03,825 : INFO : shared : created directory at : artifacts]
[2025-06-06 12:50:03,826 : INFO : shared : created directory at : artifacts/data_ingestion]
[2025-06-06 12:50:03,826 : INFO : data_ingestion : File Already exists]
[2025-06-06 12:50:03,880 : INFO : data_ingestion : Zip file extracted to artifacts/data_ingestion]
[2025-06-06 12:50:03,881 : INFO : main : >>>>> stage DATA INGESTION STAGE completed! <<<<<]
[2025-06-06 12:50:03,882 : INFO : main : >>>>> stage DATA VALIDATION STAGE started <<<<<]
[2025-06-06 12:50:03,887 : INFO : shared : Yaml File: config\config.yaml loaded successfully!]
[2025-06-06 12:50:03,889 : INFO : shared : Yaml File: params.yaml loaded successfully!]
[2025-06-06 12:50:03,892 : INFO : shared : Yaml File: schema.yaml loaded successfully!]
[2025-06-06 12:50:03,893 : INFO : shared : created directory at : artifacts]
[2025-06-06 12:50:03,894 : INFO : shared : created directory at : artifacts/data_validation]
[2025-06-06 12:50:03,935 : INFO : main : >>>>> stage DATA VALIDATION STAGE completed! <<<<<]
[2025-06-06 12:50:03,936 : INFO : main : >>>>> stage DATA VALIDATION STAGE started <<<<<]
[2025-06-06 12:50:03,937 : INFO : shared : Yaml File: config\config.yaml loaded successfully!]
[2025-06-06 12:50:03,937 : INFO : shared : Yaml File: params.yaml loaded successfully!]
[2025-06-06 12:50:03,938 : INFO : shared : Yaml File: schema.yaml loaded successfully!]
[2025-06-06 12:50:03,939 : INFO : shared : created directory at : artifacts]
[2025-06-06 12:50:03,939 : INFO : shared : created directory at : artifacts/data_transformation]
[2025-06-06 12:50:03,961 : INFO : data_transformation : Splitted data into training and test sets]
[2025-06-06 12:50:03,962 : INFO : data_transformation : The Train Data SHAPE is: (1279, 12)]
[2025-06-06 12:50:03,962 : INFO : data_transformation : The Test Data SHAPE is: (320, 12)]
[2025-06-06 12:50:03,962 : INFO : main : >>>>> stage DATA TRANSFORMATION STAGE completed! <<<<<]
[2025-06-06 12:50:03,962 : INFO : main : >>>>> stage Model Training Pipeline STAGE started <<<<<]
[2025-06-06 12:50:03,965 : INFO : shared : Yaml File: config\config.yaml loaded successfully!]
[2025-06-06 12:50:03,966 : INFO : shared : Yaml File: params.yaml loaded successfully!]
[2025-06-06 12:50:03,967 : INFO : shared : Yaml File: schema.yaml loaded successfully!]
[2025-06-06 12:50:03,968 : INFO : shared : created directory at : artifacts]
[2025-06-06 12:50:03,968 : INFO : shared : created directory at : artifacts/model_trainer]
[2025-06-06 12:50:04,022 : INFO : model_trainer : Model fit training done.]
[2025-06-06 12:50:04,023 : INFO : main : >>>>> stage Model Training Pipeline STAGE completed! <<<<<]
[2025-06-06 13:30:03,227 : INFO : shared : Yaml File: config\config.yaml loaded successfully!]
[2025-06-06 13:30:03,243 : INFO : shared : Yaml File: params.yaml loaded successfully!]
[2025-06-06 13:30:03,259 : INFO : shared : Yaml File: schema.yaml loaded successfully!]
[2025-06-06 13:30:03,261 : INFO : shared : created directory at : artifacts]
[2025-06-06 13:30:03,263 : INFO : shared : created directory at : artifacts/model_evaluation]
[2025-06-06 13:30:37,265 : INFO : shared : Yaml File: config\config.yaml loaded successfully!]
[2025-06-06 13:30:37,270 : INFO : shared : Yaml File: params.yaml loaded successfully!]
[2025-06-06 13:30:37,273 : INFO : shared : Yaml File: schema.yaml loaded successfully!]
[2025-06-06 13:30:37,275 : INFO : shared : created directory at : artifacts]
[2025-06-06 13:30:37,278 : INFO : shared : created directory at : artifacts/model_evaluation]
[2025-06-06 13:32:04,959 : INFO : shared : Yaml File: config\config.yaml loaded successfully!]
[2025-06-06 13:32:04,975 : INFO : shared : Yaml File: params.yaml loaded successfully!]
[2025-06-06 13:32:04,990 : INFO : shared : Yaml File: schema.yaml loaded successfully!]
[2025-06-06 13:32:04,993 : INFO : shared : created directory at : artifacts]
[2025-06-06 13:32:04,996 : INFO : shared : created directory at : artifacts/model_evaluation]
[2025-06-06 14:21:18,646 : INFO : shared : Yaml File: config\config.yaml loaded successfully!]
[2025-06-06 14:21:18,668 : INFO : shared : Yaml File: params.yaml loaded successfully!]
[2025-06-06 14:21:18,686 : INFO : shared : Yaml File: schema.yaml loaded successfully!]
[2025-06-06 14:21:18,689 : INFO : shared : created directory at : artifacts]
[2025-06-06 14:21:18,691 : INFO : shared : created directory at : artifacts/model_evaluation]
[2025-06-06 14:22:52,393 : INFO : shared : Yaml File: config\config.yaml loaded successfully!]
[2025-06-06 14:22:52,393 : INFO : shared : Yaml File: params.yaml loaded successfully!]
[2025-06-06 14:22:52,399 : INFO : shared : Yaml File: schema.yaml loaded successfully!]
[2025-06-06 14:22:52,402 : INFO : shared : created directory at : artifacts]
[2025-06-06 14:22:52,403 : INFO : shared : created directory at : artifacts/model_evaluation]
[2025-06-06 14:22:54,740 : INFO : shared : json file saved at: artifacts\model_evaluation\metrics.json]
[2025-06-06 14:24:10,238 : INFO : shared : Yaml File: config\config.yaml loaded successfully!]
[2025-06-06 14:24:10,238 : INFO : shared : Yaml File: params.yaml loaded successfully!]
[2025-06-06 14:24:10,248 : INFO : shared : Yaml File: schema.yaml loaded successfully!]
[2025-06-06 14:24:10,251 : INFO : shared : created directory at : artifacts]
[2025-06-06 14:24:10,253 : INFO : shared : created directory at : artifacts/model_evaluation]
[2025-06-06 14:24:10,616 : INFO : shared : json file saved at: artifacts\model_evaluation\metrics.json]
[2025-06-06 14:25:46,578 : INFO : main : >>>>> stage DATA INGESTION STAGE started <<<<<]
[2025-06-06 14:25:46,594 : INFO : shared : Yaml File: config\config.yaml loaded successfully!]
[2025-06-06 14:25:46,594 : INFO : shared : Yaml File: params.yaml loaded successfully!]
[2025-06-06 14:25:46,596 : INFO : shared : Yaml File: schema.yaml loaded successfully!]
[2025-06-06 14:25:46,596 : INFO : shared : created directory at : artifacts]
[2025-06-06 14:25:46,598 : INFO : shared : created directory at : artifacts/data_ingestion]
[2025-06-06 14:25:46,598 : INFO : data_ingestion : File Already exists]
[2025-06-06 14:25:46,608 : INFO : data_ingestion : Zip file extracted to artifacts/data_ingestion]
[2025-06-06 14:25:46,608 : INFO : main : >>>>> stage DATA INGESTION STAGE completed! <<<<<]
[2025-06-06 14:25:46,608 : INFO : main : >>>>> stage DATA VALIDATION STAGE started <<<<<]
[2025-06-06 14:25:46,608 : INFO : shared : Yaml File: config\config.yaml loaded successfully!]
[2025-06-06 14:25:46,608 : INFO : shared : Yaml File: params.yaml loaded successfully!]
[2025-06-06 14:25:46,608 : INFO : shared : Yaml File: schema.yaml loaded successfully!]
[2025-06-06 14:25:46,608 : INFO : shared : created directory at : artifacts]
[2025-06-06 14:25:46,608 : INFO : shared : created directory at : artifacts/data_validation]
[2025-06-06 14:25:46,640 : INFO : main : >>>>> stage DATA VALIDATION STAGE completed! <<<<<]
[2025-06-06 14:25:46,640 : INFO : main : >>>>> stage DATA VALIDATION STAGE started <<<<<]
[2025-06-06 14:25:46,640 : INFO : shared : Yaml File: config\config.yaml loaded successfully!]
[2025-06-06 14:25:46,655 : INFO : shared : Yaml File: params.yaml loaded successfully!]
[2025-06-06 14:25:46,655 : INFO : shared : Yaml File: schema.yaml loaded successfully!]
[2025-06-06 14:25:46,655 : INFO : shared : created directory at : artifacts]
[2025-06-06 14:25:46,655 : INFO : shared : created directory at : artifacts/data_transformation]
[2025-06-06 14:25:46,671 : INFO : data_transformation : Splitted data into training and test sets]
[2025-06-06 14:25:46,671 : INFO : data_transformation : The Train Data SHAPE is: (1279, 12)]
[2025-06-06 14:25:46,671 : INFO : data_transformation : The Test Data SHAPE is: (320, 12)]
[2025-06-06 14:25:46,679 : INFO : main : >>>>> stage DATA TRANSFORMATION STAGE completed! <<<<<]
[2025-06-06 14:25:46,679 : INFO : main : >>>>> stage Model Training Pipeline STAGE started <<<<<]
[2025-06-06 14:25:46,679 : INFO : shared : Yaml File: config\config.yaml loaded successfully!]
[2025-06-06 14:25:46,679 : INFO : shared : Yaml File: params.yaml loaded successfully!]
[2025-06-06 14:25:46,679 : INFO : shared : Yaml File: schema.yaml loaded successfully!]
[2025-06-06 14:25:46,679 : INFO : shared : created directory at : artifacts]
[2025-06-06 14:25:46,679 : INFO : shared : created directory at : artifacts/model_trainer]
[2025-06-06 14:25:46,750 : INFO : model_trainer : Model fit training done.]
[2025-06-06 14:25:46,750 : INFO : main : >>>>> stage Model Training Pipeline STAGE completed! <<<<<]
[2025-06-06 14:28:55,052 : INFO : main : >>>>> stage DATA INGESTION PIPELINE STAGE started <<<<<]
[2025-06-06 14:28:55,052 : INFO : shared : Yaml File: config\config.yaml loaded successfully!]
[2025-06-06 14:28:55,058 : INFO : shared : Yaml File: params.yaml loaded successfully!]
[2025-06-06 14:28:55,058 : INFO : shared : Yaml File: schema.yaml loaded successfully!]
[2025-06-06 14:28:55,058 : INFO : shared : created directory at : artifacts]
[2025-06-06 14:28:55,058 : INFO : shared : created directory at : artifacts/data_ingestion]
[2025-06-06 14:28:55,058 : INFO : data_ingestion : File Already exists]
[2025-06-06 14:28:55,058 : INFO : data_ingestion : Zip file extracted to artifacts/data_ingestion]
[2025-06-06 14:28:55,058 : INFO : main : >>>>> stage DATA INGESTION PIPELINE STAGE completed! <<<<<]
[2025-06-06 14:28:55,058 : INFO : main : >>>>> stage DATA VALIDATION PIPELINE STAGE started <<<<<]
[2025-06-06 14:28:55,058 : INFO : shared : Yaml File: config\config.yaml loaded successfully!]
[2025-06-06 14:28:55,058 : INFO : shared : Yaml File: params.yaml loaded successfully!]
[2025-06-06 14:28:55,058 : INFO : shared : Yaml File: schema.yaml loaded successfully!]
[2025-06-06 14:28:55,058 : INFO : shared : created directory at : artifacts]
[2025-06-06 14:28:55,058 : INFO : shared : created directory at : artifacts/data_validation]
[2025-06-06 14:28:55,117 : INFO : main : >>>>> stage DATA VALIDATION PIPELINE STAGE completed! <<<<<]
[2025-06-06 14:28:55,117 : INFO : main : >>>>> stage DATA VALIDATION PIPELINE STAGE started <<<<<]
[2025-06-06 14:28:55,121 : INFO : shared : Yaml File: config\config.yaml loaded successfully!]
[2025-06-06 14:28:55,121 : INFO : shared : Yaml File: params.yaml loaded successfully!]
[2025-06-06 14:28:55,121 : INFO : shared : Yaml File: schema.yaml loaded successfully!]
[2025-06-06 14:28:55,121 : INFO : shared : created directory at : artifacts]
[2025-06-06 14:28:55,121 : INFO : shared : created directory at : artifacts/data_transformation]
[2025-06-06 14:28:55,144 : INFO : data_transformation : Splitted data into training and test sets]
[2025-06-06 14:28:55,144 : INFO : data_transformation : The Train Data SHAPE is: (1279, 12)]
[2025-06-06 14:28:55,144 : INFO : data_transformation : The Test Data SHAPE is: (320, 12)]
[2025-06-06 14:28:55,144 : INFO : main : >>>>> stage DATA TRANSFORMATION PIPELINE STAGE completed! <<<<<]
[2025-06-06 14:28:55,144 : INFO : main : >>>>> stage Model Training PIPELINE STAGE started <<<<<]
[2025-06-06 14:28:55,144 : INFO : shared : Yaml File: config\config.yaml loaded successfully!]
[2025-06-06 14:28:55,144 : INFO : shared : Yaml File: params.yaml loaded successfully!]
[2025-06-06 14:28:55,144 : INFO : shared : Yaml File: schema.yaml loaded successfully!]
[2025-06-06 14:28:55,144 : INFO : shared : created directory at : artifacts]
[2025-06-06 14:28:55,144 : INFO : shared : created directory at : artifacts/model_trainer]
[2025-06-06 14:28:55,219 : INFO : model_trainer : Model fit training done.]
[2025-06-06 14:28:55,219 : INFO : main : >>>>> stage Model Training PIPELINE STAGE completed! <<<<<]
[2025-06-06 14:28:55,219 : INFO : main : >>>>> stage Model Evaluation PIPELINE STAGE  started <<<<<]
[2025-06-06 14:28:55,225 : INFO : shared : Yaml File: config\config.yaml loaded successfully!]
[2025-06-06 14:28:55,225 : INFO : shared : Yaml File: params.yaml loaded successfully!]
[2025-06-06 14:28:55,225 : INFO : shared : Yaml File: schema.yaml loaded successfully!]
[2025-06-06 14:28:55,225 : INFO : shared : created directory at : artifacts]
[2025-06-06 14:28:55,225 : INFO : shared : created directory at : artifacts/model_evaluation]
[2025-06-06 14:28:55,225 : ERROR : main : 'ModelEvaluation' object has no attribute 'train']
Traceback (most recent call last):
  File "C:\ML_OPS\Projects\project1\main.py", line 44, in <module>
    model_evaluation.initiate_model_evaluation()
  File "C:\ML_OPS\Projects\project1\src\DataScienceProject\pipeline\model_evaluation_pipeline.py", line 15, in initiate_model_evaluation
    model_evaluation.train()
AttributeError: 'ModelEvaluation' object has no attribute 'train'
[2025-06-06 14:42:07,241 : INFO : main : >>>>> stage DATA INGESTION PIPELINE STAGE started <<<<<]
[2025-06-06 14:42:07,283 : INFO : shared : Yaml File: config\config.yaml loaded successfully!]
[2025-06-06 14:42:07,305 : INFO : shared : Yaml File: params.yaml loaded successfully!]
[2025-06-06 14:42:07,330 : INFO : shared : Yaml File: schema.yaml loaded successfully!]
[2025-06-06 14:42:07,330 : INFO : shared : created directory at : artifacts]
[2025-06-06 14:42:07,330 : INFO : shared : created directory at : artifacts/data_ingestion]
[2025-06-06 14:42:07,330 : INFO : data_ingestion : File Already exists]
[2025-06-06 14:42:07,390 : INFO : data_ingestion : Zip file extracted to artifacts/data_ingestion]
[2025-06-06 14:42:07,390 : INFO : main : >>>>> stage DATA INGESTION PIPELINE STAGE completed! <<<<<]
[2025-06-06 14:42:07,390 : INFO : main : >>>>> stage DATA VALIDATION PIPELINE STAGE started <<<<<]
[2025-06-06 14:42:07,398 : INFO : shared : Yaml File: config\config.yaml loaded successfully!]
[2025-06-06 14:42:07,400 : INFO : shared : Yaml File: params.yaml loaded successfully!]
[2025-06-06 14:42:07,400 : INFO : shared : Yaml File: schema.yaml loaded successfully!]
[2025-06-06 14:42:07,400 : INFO : shared : created directory at : artifacts]
[2025-06-06 14:42:07,400 : INFO : shared : created directory at : artifacts/data_validation]
[2025-06-06 14:42:07,449 : INFO : main : >>>>> stage DATA VALIDATION PIPELINE STAGE completed! <<<<<]
[2025-06-06 14:42:07,449 : INFO : main : >>>>> stage DATA VALIDATION PIPELINE STAGE started <<<<<]
[2025-06-06 14:42:07,449 : INFO : shared : Yaml File: config\config.yaml loaded successfully!]
[2025-06-06 14:42:07,449 : INFO : shared : Yaml File: params.yaml loaded successfully!]
[2025-06-06 14:42:07,449 : INFO : shared : Yaml File: schema.yaml loaded successfully!]
[2025-06-06 14:42:07,449 : INFO : shared : created directory at : artifacts]
[2025-06-06 14:42:07,449 : INFO : shared : created directory at : artifacts/data_transformation]
[2025-06-06 14:42:07,476 : INFO : data_transformation : Splitted data into training and test sets]
[2025-06-06 14:42:07,476 : INFO : data_transformation : The Train Data SHAPE is: (1279, 12)]
[2025-06-06 14:42:07,476 : INFO : data_transformation : The Test Data SHAPE is: (320, 12)]
[2025-06-06 14:42:07,479 : INFO : main : >>>>> stage DATA TRANSFORMATION PIPELINE STAGE completed! <<<<<]
[2025-06-06 14:42:07,479 : INFO : main : >>>>> stage Model Training PIPELINE STAGE started <<<<<]
[2025-06-06 14:42:07,480 : INFO : shared : Yaml File: config\config.yaml loaded successfully!]
[2025-06-06 14:42:07,480 : INFO : shared : Yaml File: params.yaml loaded successfully!]
[2025-06-06 14:42:07,480 : INFO : shared : Yaml File: schema.yaml loaded successfully!]
[2025-06-06 14:42:07,480 : INFO : shared : created directory at : artifacts]
[2025-06-06 14:42:07,480 : INFO : shared : created directory at : artifacts/model_trainer]
[2025-06-06 14:42:07,536 : INFO : model_trainer : Model fit training done.]
[2025-06-06 14:42:07,536 : INFO : main : >>>>> stage Model Training PIPELINE STAGE completed! <<<<<]
[2025-06-06 14:42:07,536 : INFO : main : >>>>> stage Model Evaluation PIPELINE STAGE  started <<<<<]
[2025-06-06 14:42:07,542 : INFO : shared : Yaml File: config\config.yaml loaded successfully!]
[2025-06-06 14:42:07,542 : INFO : shared : Yaml File: params.yaml loaded successfully!]
[2025-06-06 14:42:07,543 : INFO : shared : Yaml File: schema.yaml loaded successfully!]
[2025-06-06 14:42:07,543 : INFO : shared : created directory at : artifacts]
[2025-06-06 14:42:07,543 : INFO : shared : created directory at : artifacts/model_evaluation]
[2025-06-06 14:42:08,618 : INFO : shared : json file saved at: artifacts\model_evaluation\metrics.json]
[2025-06-06 14:42:08,618 : ERROR : main : Invalid value "rmse: " for parameter 'name' supplied: Names may only contain alphanumerics, underscores (_), dashes (-), periods (.), spaces ( ) and slashes (/).]
Traceback (most recent call last):
  File "C:\ML_OPS\Projects\project1\main.py", line 44, in <module>
    model_evaluation.initiate_model_evaluation()
  File "C:\ML_OPS\Projects\project1\src\DataScienceProject\pipeline\model_evaluation_pipeline.py", line 15, in initiate_model_evaluation
    model_evaluation.log_into_mlflow()
  File "C:\ML_OPS\Projects\project1\src\DataScienceProject\components\model_evaluationn.py", line 46, in log_into_mlflow
    mlflow.log_metric('rmse: ',rmse)
  File "C:\ML_OPS\Projects\project1\myvenv\lib\site-packages\mlflow\tracking\fluent.py", line 923, in log_metric
    return MlflowClient().log_metric(
  File "C:\ML_OPS\Projects\project1\myvenv\lib\site-packages\mlflow\tracking\client.py", line 1972, in log_metric
    return self._tracking_client.log_metric(
  File "C:\ML_OPS\Projects\project1\myvenv\lib\site-packages\mlflow\tracking\_tracking_service\client.py", line 672, in log_metric
    self.store.log_metric(run_id, metric)
  File "C:\ML_OPS\Projects\project1\myvenv\lib\site-packages\mlflow\store\tracking\file_store.py", line 982, in log_metric
    _validate_metric(metric.key, metric.value, metric.timestamp, metric.step)
  File "C:\ML_OPS\Projects\project1\myvenv\lib\site-packages\mlflow\utils\validation.py", line 206, in _validate_metric
    _validate_metric_name(key, append_to_json_path(path, "name"))
  File "C:\ML_OPS\Projects\project1\myvenv\lib\site-packages\mlflow\utils\validation.py", line 181, in _validate_metric_name
    raise MlflowException(
mlflow.exceptions.MlflowException: Invalid value "rmse: " for parameter 'name' supplied: Names may only contain alphanumerics, underscores (_), dashes (-), periods (.), spaces ( ) and slashes (/).
[2025-06-06 14:42:48,718 : INFO : main : >>>>> stage DATA INGESTION PIPELINE STAGE started <<<<<]
[2025-06-06 14:42:48,720 : INFO : shared : Yaml File: config\config.yaml loaded successfully!]
[2025-06-06 14:42:48,722 : INFO : shared : Yaml File: params.yaml loaded successfully!]
[2025-06-06 14:42:48,724 : INFO : shared : Yaml File: schema.yaml loaded successfully!]
[2025-06-06 14:42:48,724 : INFO : shared : created directory at : artifacts]
[2025-06-06 14:42:48,724 : INFO : shared : created directory at : artifacts/data_ingestion]
[2025-06-06 14:42:48,726 : INFO : data_ingestion : File Already exists]
[2025-06-06 14:42:48,726 : INFO : data_ingestion : Zip file extracted to artifacts/data_ingestion]
[2025-06-06 14:42:48,726 : INFO : main : >>>>> stage DATA INGESTION PIPELINE STAGE completed! <<<<<]
[2025-06-06 14:42:48,726 : INFO : main : >>>>> stage DATA VALIDATION PIPELINE STAGE started <<<<<]
[2025-06-06 14:42:48,726 : INFO : shared : Yaml File: config\config.yaml loaded successfully!]
[2025-06-06 14:42:48,732 : INFO : shared : Yaml File: params.yaml loaded successfully!]
[2025-06-06 14:42:48,732 : INFO : shared : Yaml File: schema.yaml loaded successfully!]
[2025-06-06 14:42:48,732 : INFO : shared : created directory at : artifacts]
[2025-06-06 14:42:48,732 : INFO : shared : created directory at : artifacts/data_validation]
[2025-06-06 14:42:48,772 : INFO : main : >>>>> stage DATA VALIDATION PIPELINE STAGE completed! <<<<<]
[2025-06-06 14:42:48,772 : INFO : main : >>>>> stage DATA VALIDATION PIPELINE STAGE started <<<<<]
[2025-06-06 14:42:48,779 : INFO : shared : Yaml File: config\config.yaml loaded successfully!]
[2025-06-06 14:42:48,779 : INFO : shared : Yaml File: params.yaml loaded successfully!]
[2025-06-06 14:42:48,779 : INFO : shared : Yaml File: schema.yaml loaded successfully!]
[2025-06-06 14:42:48,779 : INFO : shared : created directory at : artifacts]
[2025-06-06 14:42:48,779 : INFO : shared : created directory at : artifacts/data_transformation]
[2025-06-06 14:42:48,795 : INFO : data_transformation : Splitted data into training and test sets]
[2025-06-06 14:42:48,795 : INFO : data_transformation : The Train Data SHAPE is: (1279, 12)]
[2025-06-06 14:42:48,795 : INFO : data_transformation : The Test Data SHAPE is: (320, 12)]
[2025-06-06 14:42:48,795 : INFO : main : >>>>> stage DATA TRANSFORMATION PIPELINE STAGE completed! <<<<<]
[2025-06-06 14:42:48,795 : INFO : main : >>>>> stage Model Training PIPELINE STAGE started <<<<<]
[2025-06-06 14:42:48,795 : INFO : shared : Yaml File: config\config.yaml loaded successfully!]
[2025-06-06 14:42:48,795 : INFO : shared : Yaml File: params.yaml loaded successfully!]
[2025-06-06 14:42:48,795 : INFO : shared : Yaml File: schema.yaml loaded successfully!]
[2025-06-06 14:42:48,795 : INFO : shared : created directory at : artifacts]
[2025-06-06 14:42:48,795 : INFO : shared : created directory at : artifacts/model_trainer]
[2025-06-06 14:42:48,879 : INFO : model_trainer : Model fit training done.]
[2025-06-06 14:42:48,879 : INFO : main : >>>>> stage Model Training PIPELINE STAGE completed! <<<<<]
[2025-06-06 14:42:48,883 : INFO : main : >>>>> stage Model Evaluation PIPELINE STAGE  started <<<<<]
[2025-06-06 14:42:48,888 : INFO : shared : Yaml File: config\config.yaml loaded successfully!]
[2025-06-06 14:42:48,888 : INFO : shared : Yaml File: params.yaml loaded successfully!]
[2025-06-06 14:42:48,888 : INFO : shared : Yaml File: schema.yaml loaded successfully!]
[2025-06-06 14:42:48,888 : INFO : shared : created directory at : artifacts]
[2025-06-06 14:42:48,888 : INFO : shared : created directory at : artifacts/model_evaluation]
[2025-06-06 14:42:49,742 : INFO : shared : json file saved at: artifacts\model_evaluation\metrics.json]
[2025-06-06 14:43:07,124 : INFO : main : >>>>> stage Model Evaluation PIPELINE STAGE  completed! <<<<<]
